src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -174,7 +174,10 @@ function shouldNotKeepSorted_HeatmapTracksHelper(
     // Check if
     // (1) A track added/removed
     // (2) Track data changed
-    if (differentTracksOrChangedData(nextProps.geneticTracks || [], prevProps.geneticTracks || [])) {
+    if (
+        differentTracksOrChangedData(nextProps.heatmapTracks, prevProps.heatmapTracks || [] )
```

Comment:
wow I really f*cked this up! :D

Reply:
At least it was only a little performance thingy up until I built heatmap tracks that jump around to different track groups 👼